diff --git a/third_party/hidapi/hidapi_linux.c b/third_party/hidapi/hidapi_linux.c
index 38f6753..21d04c0 100644
--- a/third_party/hidapi/hidapi_linux.c
+++ b/third_party/hidapi/hidapi_linux.c
@@ -305,7 +305,7 @@ static int get_device_string(hid_device *dev, enum device_string_id key, wchar_t
 					const char *str;
 					const char *key_str = NULL;
 
-					if (key >= 0 && key < DEVICE_STRING_COUNT) {
+					if (key < DEVICE_STRING_COUNT) {
 						key_str = device_string_names[key];
 					} else {
 						ret = -1;
